frontend/model/contracts/voting/rules.js

```diff
@@ -2,6 +2,10 @@
 
 import { literalOf, unionOf } from '~/frontend/utils/flowTyper.js'
 
+// Error: Circular dependency:
+// import { PROPOSAL_REMOVE_MEMBER } from '@model/contracts/voting/proposals.js'
+const PROPOSAL_REMOVE_MEMBER = 'remove-member'
```

Comment:
TODO - Where should we create the file with all these `const`?

Reply:
How about under `voting/constants.js` ?